{"name": "check /reconciliation/user_data/", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\reconciliation_I_upload.py\", line 160, in step_impl\n    assert context.ac_template==context.exp_template\n"}, "steps": [{"name": "Given create expected template of user_data", "status": "passed", "start": 1611571215402, "stop": 1611571215402}, {"name": "And get actual template of user_data", "status": "passed", "start": 1611571215403, "stop": 1611571215511}, {"name": "Then compare actual and expected templates", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\wsu\\pycharmprojects\\msw_api\\venv\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\reconciliation_I_upload.py\", line 160, in step_impl\n    assert context.ac_template==context.exp_template\n"}, "start": 1611571215511, "stop": 1611571215516}], "start": 1611571215400, "stop": 1611571215517, "uuid": "48be6b30-b20d-4d34-ac2d-78573dfde91b", "historyId": "a25d58092254c90c0631e0b574751eb3", "fullName": "features/II-Reconciliation/II-reconciliation_upload.feature:check /reconciliation/user_data/", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "manager_user"}, {"name": "tag", "value": "super_user"}, {"name": "feature", "value": "reconciliation upload"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}